bcelndora.module

```diff
@@ -174,3 +174,10 @@ function bcelndora_form_xml_form_builder_ingest_form_alter(&$form, &$form_state,
     }
   }
 }
+
+/**
+ * Implements hook_form_islandora_solr_simple_search_form_alter().
+ */
+function bcelndora_form_islandora_solr_simple_search_form_alter(&$form, &$form_state, $form_id) {
+  unset($form['simple']['islandora_simple_search_query']['#title']);
```

Comment:
Instead of `unset()`ing, we could be nice for screen readers and just set it to invisible with the [`#title_display` property](https://api.drupal.org/api/drupal/developer!topics!forms_api_reference.html/7.x#title_display).